packages/react-components/react-dialog/src/utils/useDisableBodyScroll.ts

```diff
@@ -0,0 +1,44 @@
+import { makeStaticStyles } from '@griffel/react';
+
+const disableScrollingClassName = 'disableScrolling' as const;
+const fluentDisableScrollCountProp = '__fluentDisableScrollCount' as const;
+
+const useStaticStyles = makeStaticStyles({
+  [`.${disableScrollingClassName}`]: {
+    overflowY: 'hidden',
+  },
+});
+
+type FluentDisableScrollElement = HTMLElement & {
+  [fluentDisableScrollCountProp]: number;
+};
+
+export function useDisableScroll() {
+  useStaticStyles();
```

Comment:
As we discussed in offline: `makeStaticStyles` does not have support for build time optimisations, so I would avoid using. Actually for that reason we see +8kb in bundle size 😭 

![image](https://user-images.githubusercontent.com/14183168/184978529-e7899cb6-900c-4e94-99df-1cf0ca9a9c84.png)